twine/utils.py

```diff
@@ -210,25 +210,27 @@ def get_userpass_value(
     key: str,
     prompt_strategy: Optional[Callable[[], str]] = None,
 ) -> Optional[str]:
-    """Get the username / password from config.
+    """Get a credential (e.g. a username or password) from the configuration.
 
     Uses the following rules:
 
-    1. If it is specified on the cli (`cli_value`), use that.
-    2. If `config[key]` is specified, use that.
-    3. If `prompt_strategy`, prompt using `prompt_strategy`.
-    4. Otherwise return None
-
-    :param cli_value: The value supplied from the command line or `None`.
-    :type cli_value: unicode or `None`
-    :param config: Config dictionary
-    :type config: dict
-    :param key: Key to find the config value.
-    :type key: unicode
-    :prompt_strategy: Argumentless function to return fallback value.
-    :type prompt_strategy: function
-    :returns: The value for the username / password
-    :rtype: unicode
+    1. If ``cli_value`` is specified, use that.
+    2. If ``config[key]`` is specified, use that.
+    3. If ``prompt_strategy`` is specified, use its return value.
+    4. Otherwise return ``None``
+
+    :param cli_value:
+        The value supplied from the command line.
+    :param config:
+        A dictionary of repository configuration values.
+    :param key:
+        The credential to look up in ``config``, e.g. ``"username"`` or ``"password"``.
+    :param prompt_strategy:
+        An argumentless function to get the value, e.g. from keyring or by prompting
+        the user.
+
+    :return:
```

Comment:
Is it `:return:` or `:returns:`? I thought it was always the latter

Reply:
According to https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html, it is `:return:`.

Reply:
I've always seen and used `:returns:` and it works so I wonder if it is just an alias